src/Heading/index.js

```diff
@@ -3,21 +3,22 @@ import * as React from "react";
 import * as tokens from "@kiwicom/orbit-design-tokens";
 
 const weightHeading = {
-  medium: tokens.fontWeightMedium,
-  bold: tokens.fontWeightBold,
+  display: tokens.fontWeightHeadingDisplay,
+  title1: tokens.fontWeightHeadingLevel1,
+  title2: tokens.fontWeightHeadingLevel2,
+  title3: tokens.fontWeightHeadingLevel3,
 };
 
 const sizeHeading = {
   display: tokens.fontSizeHeadingDisplay,
-  large: tokens.fontSizeHeadingLarge,
-  medium: tokens.fontSizeHeadingMedium,
-  small: tokens.fontSizeHeadingSmall,
+  title1: tokens.fontSizeHeadingTitle1,
+  title2: tokens.fontSizeHeadingTitle2,
+  title3: tokens.fontSizeHeadingTitle3,
 };
 
 type Props = {
   element: "h1" | "h2" | "h3" | "h4" | "h5",
-  size: $Keys<typeof sizeHeading>,
-  weight: $Keys<typeof weightHeading>,
+  type: $Keys<typeof weightHeading>,
   children: React.Node,
   className?: string,
```

Comment:
Is `className` really necessary ? I thought we talked about wrapping component with classname and let component without classnames

Reply:
Will fix this